docs/Using Price Feeds/get-the-latest-price.html

```diff
@@ -2,43 +2,43 @@
 layout: nodes.liquid
 section: smartContract
 date: Last Modified
-title: "Consuming Price Feeds"
+title: "Consuming Data Feeds"
 permalink: "docs/get-the-latest-price/"
-whatsnext: {"Historical Price Data":"/docs/historical-price-data/", "Price Feeds API Reference":"/docs/price-feeds-api-reference/", "Price Feeds Contract Addresses":"/docs/reference-contracts/"}
+whatsnext: {"Historical Price Data":"/docs/historical-price-data/", "Data Feeds API Reference":"/docs/price-feeds-api-reference/", "Data Feeds Contract Addresses":"/docs/reference-contracts/"}
 metadata:
-  title: "Consuming Price Feeds"
-  description: "How to use Chainlink Price Feeds in your smart contracts."
+  title: "Consuming Data Feeds"
+  description: "How to use Chainlink Data Feeds in your smart contracts."
   image:
     0: "/files/OpenGraph_V3.png"
 ---
  {% markdown %}
 
 Price data feeds are available on the following networks:
 - [EVM-compatible networks](#ethereum-network-examples)
-- [Solana networks](#solana-network-examples)
+- [Solana network](#solana-network-examples)
 
-To get a full list of supported blockchains for price feeds, see the [Price Feeds Contract Addresses](../reference-contracts/) page.
+To get a full list of supported blockchains for data feeds, see the [Data Feeds Contract Addresses](../reference-contracts/) page.
 
 > 📘 Note
 >
-> The full list of price feeds for each network are available from [Price Feed Contracts](../reference-contracts/).
+> The full list of data feeds for each network are available from [Data Feed Contracts](../reference-contracts/).
 
 > 📘 New Feed Registry
-> You can now use the [Feed Registry](../feed-registry/) to reference price feed assets by name or currency identifier instead of pair/proxy address.
+> You can now use the [Feed Registry](../feed-registry/) to reference data feed assets by name or currency identifier instead of pair/proxy address.
```

Comment:
```suggestion
> You can now use the [Feed Registry](../feed-registry/) to reference data feed assets by name or identifier instead of pair/proxy address.
```